apps/web/src/features/cohorts/components/CohortForm.tsx

```
@@ -1,867 +1,752 @@
+                  );
+                }
+
+                return sessionResponse.json();
+              } catch (error) {
+                const errorMessage =
+                  error instanceof Error
+                    ? error.message
+                    : `Unknown error for session ${
+                        session.id || `temp-${index}`
+                      }`;
+                console.error(`Failed to save weekly session:`, {
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)